src/lib/types.ts

```diff
@@ -52,13 +54,44 @@ export interface Coordinates {
 }
 
 export type ViewportCoordinates = Coordinates;
-export type CanvasCoordinates = Coordinates;
+export type CanvasCoordinate = Coordinates;
```

Comment:
```suggestion
export type CanvasCoordinates = Coordinates;
```
.. og de andre steder den er brugt